mozzi_config.h

```diff
@@ -89,5 +89,16 @@ You need to have \#define STEREO_HACK true in mozzi_config.h
 */
 #define STEREO_HACK false
 
+
+/**
+  Uncomment this line if you want to use Mozzi with an external DAC of type MCP4922. Note that you will need the MCP49XX library (see AudioConfigMCP4922 for details.
+*/
+//#define __DAC_MCP__
```

Comment:
To follow the style of existing defines, remove the __ around the name. Also, I'd suggest to name it generically ("EXTERNAL_DAC", or something). The details (exact chip) would then be configured in AudioConfigExternalDAC.h.